pkg/daemon/controller.go

```diff
@@ -1270,6 +1273,31 @@ func (c *Controller) deleteSubnetQos(subnet *kubeovnv1.Subnet) error {
 	return nil
 }
 
+func (c *Controller) operatemod() {
```

Comment:
should use camelcase in all function and variable name

Reply:
all variable and function names are renamed...